lib/mpool/src/mpool.c

```diff
@@ -224,12 +234,14 @@ mpool_create(const char *home, const struct mpool_cparams *cparams)
     for (i = MP_MED_BASE; i < MP_MED_COUNT; i++) {
         struct mclass_params mcp = {0};
 
-        /* If capacity path is the default, automatically create it */
-        if (i == MP_MED_CAPACITY) {
+        /* If the capacity/pmem path is the default, automatically create it */
+        if (i == MP_MED_CAPACITY || i == MP_MED_PMEM) {
```

Comment:
Should we only create the pmem default path if it is a pmem-only db?

Reply:
This code creates the default "capacity" or "pmem" directories only if the KVDB layer instructs it to do so..

Regarding whether we should have a pmem default only for a pmem-only KVDB,  I thought about it for some time earlier today but couldn't see the value in a user wanting to place only a KVDB home in a DAX filesystem but not want a pmem media class for this KVDB. I will put some more thinking into it and get back.